ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml

```diff
@@ -73,6 +73,29 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-ovn-dualstack
+- as: e2e-metal-ipi-ovn-dualstack-v6
+  cluster: build05
+  optional: true
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6v4
+      TEST_SKIPS: should run iperf2 \| CVE-2021-29923 \| pod containing more than
```

Comment:
@cybertron are these skips meant to be temporary? If not I'd expected them to be filtered in `openshift/origin` (ie https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go)

Reply:
Yes, they're temporary. I have a story on our backlog to look into why they're failing, but I expect we should fix all of them eventually.

https://issues.redhat.com/browse/OPNET-337